server.cjs

```diff
@@ -0,0 +1,63 @@
+const http = require('http')
+const path = require('path')
+const fs = require('fs')
+const timezone = require('./src/timeZone.cjs')
+let allCityData = ''
+http.createServer(function (req, res) {
+  const reqUrl = req.url
+  let contentType = 'text/html'
+  let filePath = '.' + reqUrl
+  if (filePath === './') {
+    filePath = './src/index.html'
+  }
+  const extName = path.extname(filePath)
+  switch (extName) {
+    case '.css':
+      filePath = './src' + reqUrl
+      contentType = 'text/css'
+      break
+    case '.js':
+      filePath = './src' + reqUrl
+      contentType = 'text/javascript'
+      break
+    case '.svg':
+      contentType = 'image/svg+xml'
+      break
+    case '.ico' :
+      filePath = '/Assets/HTML&CSS/Weather_Icons/cloudyIcon.svg'
+      contentType = 'image/svg+xml'
+      break
+    case '.png':
```

Comment:
If .png file type is not needed let's not have a case for it.
Also let's add a default case.